databrowser/src/layouts/menu/OveralyMenuList.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div class="flex flex-col w-full">
+    <button class="inline-flex items-center" @click="$emit('arrow-back')">
+      <ArrowLeft
+        v-if="showArrow"
+        :title="$t('header.menu.goBackAction')"
+        class="md:hidden"
+      />
+      <span class="py-2 pl-2 text-xl font-semibold">{{ item.label }}</span>
+    </button>
+    <ul class="md:max-w-xs min-w-[20rem]">
+      <li
+        v-for="(category, index) in item.categories"
+        :key="category.label"
+        :class="[
+          selected == index ? 'bg-green-500 bg-opacity-10' : 'hover:bg-gray-50',
+        ]"
+        class="flex whitespace-nowrap"
+      >
+        <button
+          v-if="isMenuCategory(category) && category.categories.length"
+          class="
+            inline-flex
+            flex-1
+            justify-between
+            items-center
+            py-2
+            px-4
+            w-full
+            md:max-w-xs
+            text-left
+          "
+          @click="setSelected(category, index)"
+        >
+          <span>{{ category.label }}</span>
+          <ArrowRight />
+        </button>
+        <router-link
+          v-if="isMenuLink(category)"
+          :to="category.url"
+          class="flex-1 py-2 px-4 w-full text-left"
+          @click="$emit('close-dialog')"
+        >
+          {{ category.label }}
+        </router-link>
+      </li>
+    </ul>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from '@vue/runtime-core';
+import ArrowRight from '../../components/svg/ArrowRight.vue';
+import { ref, watch } from 'vue';
+import ArrowLeft from '../../components/svg/ArrowLeft.vue';
+
+export type MenuLink = {
+  label: string;
+  url: string;
+};
+
+export type MenuCategory = {
```

Comment:
What do you think about calling this `MenuColumn`? And the field `categories` could be called `items`, because there can also be links not just categories. What do you think?